tests/core/data_structures/test_networks.py

```
@@ -35,7 +35,108 @@
 from mqt.yaqs.core.data_structures.simulation_parameters import Observable
-from mqt.yaqs.core.libraries.gate_library import GateLibrary, Id, X, Y, Z
+from mqt.yaqs.core.libraries.gate_library import GateLibrary, Id, X, Z
+
+# ---- single-qubit ops ----
+_I2 = np.eye(2, dtype=complex)
+_X2 = np.array([[0, 1], [1, 0]], dtype=complex)
+_Y2 = np.array([[0, -1j], [1j, 0]], dtype=complex)
+_Z2 = np.array([[1, 0], [0, -1]], dtype=complex)
+
+
+def _embed_one_body(op: np.ndarray, L: int, i: int) -> np.ndarray:  # noqa: N803
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `N803`)

Remove unused `noqa` directive

(RUF100)